### DISCUSSION TITLE
Cant install ubuntu server 19.04

### DISCUSSION THREAD

**Discussion by c001os** [strapi/strapi#5945] [Legacy Topics]
**Describe the bug**
Cant install, ubuntu server 19.04

**Steps to reproduce the behavior**
yarn create strapi-app strapi

**Expected behavior**
Installation fails

**Screenshots**

**Code snippets**
Error while installing dependencies:
warning strapi > boom@7.3.0: This module has moved and is now available at @hapi/boom. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
warning strapi > boom > hoek@6.1.3: This module has moved and is now available at @hapi/hoek. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
warning strapi-admin > webpack-dev-server > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
warning strapi-admin > webpack-dev-server > chokidar > fsevents@1.2.12: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
warning strapi-admin > @babel/polyfill > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning strapi-admin > react-intl > intl-relativeformat@2.2.0: This package has been deprecated, please see migration guide at 'https://github.com/formatjs/formatjs/tree/master/packages/intl-relativeformat#migration-guide'
warning strapi-admin > react-virtualized > babel-runtime > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning strapi-admin > webpack > watchpack > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
warning strapi-admin > react-datetime > create-react-class > fbjs > core-js@1.2.7: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning strapi-admin > reactstrap > react-popper > popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1
warning strapi-admin > strapi-helper-plugin > reactstrap > react-popper > popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1
warning strapi-admin > webpack-dev-server > chokidar > braces > snapdragon > source-map-resolve > resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
warning strapi-admin > webpack-dev-server > chokidar > braces > snapdragon > source-map-resolve > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
warning strapi-plugin-users-permissions > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
warning strapi-plugin-email > strapi-provider-email-sendmail > sendmail > mailcomposer@3.12.0: This project is unmaintained
warning strapi-plugin-email > strapi-provider-email-sendmail > sendmail > mailcomposer > buildmail@3.10.0: This project is unmaintained
warning sqlite3 > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
warning "strapi-admin > bootstrap@4.4.1" has unmet peer dependency "jquery@1.9.1 - 3".
warning "strapi-admin > bootstrap@4.4.1" has unmet peer dependency "popper.js@^1.16.0".
warning "strapi-admin > @buffetjs/core > react-dates@21.8.0" has unmet peer dependency "react-with-direction@^1.3.1".
warning "strapi-plugin-users-permissions > grant-koa@4.7.0" has unmet peer dependency "koa@>=0.x.x".
warning " > strapi-connector-bookshelf@3.0.0-beta.20" has incorrect peer dependency "knex@^0.20.0".
error /var/www/strapi/node_modules/sharp: Command failed.
Exit code: 1
Command: (node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)
Arguments:
Directory: /var/www/strapi/node_modules/sharp

**System**
- Node.js version: v12.16.2
- NPM version: 6.14.4
- Strapi version:  3.0.0-beta.20
- Database: mysql
- Operating system: ubuntu server 19.04

**Additional context**

**Comment by derrickmehaffy** [comment]
Can you please make your massive wall of text readable?

**Comment by c001os** [comment]
> Can you please make your massive wall of text readable?

Sorry, updated, thank you!

**Comment by c001os** [comment]
Additionaly:

npm -i fails too:

`npm i strapi
npm WARN deprecated boom@7.3.0: This module has moved and is now available at @hapi/boom. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
npm WARN deprecated hoek@6.1.3: This module has moved and is now available at @hapi/hoek. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! Unexpected token  in JSON at position 4533 while parsing near '...rt":"~0.3.0"},"disb�/���"���gW�e211f3...'`

**Comment by derrickmehaffy** [comment]
`node-gyp rebuild` 

That's a python problem (node-gyp is a python based build tool)

Do you have the package `build-essential` installed?

**Comment by derrickmehaffy** [comment]
As a side point, if you are using root you shouldn't be......

**Reply by derrickmehaffy** [reply]
Also moving this over to a discussion as it's not a bug with Strapi but environment problem.

**Reply by derrickmehaffy** [reply]
As ubuntu 19.04 is no longer going to be supported (by Canonical) and I feel this might be a user environment error with whom the user hasn't responded, I'm going to mark this as answered. If anyone else hits this problem please open up a new discussion.
